lib/game_teams_manager.rb

```diff
@@ -56,28 +56,28 @@ def teams_sort_by_average_goal
       team_average_goals(team.team_id)
     end
   end
-end
 
-#   def find_all_away_teams
-#      @game_teams_array.find_all do |gameteam|
-#       gameteam.hoa == "away"
-#     end
-#   end
+  def find_all_away_teams
+     @game_teams_array.find_all do |gameteam|
+      gameteam.hoa == "away"
+    end
+  end
 
-#   def away_games_by_team_id
+  def away_games_by_team_id
 
-#     find_all_away_teams.group_by do |game|
-#       game.team_id
-#     end
-#   end
+    find_all_away_teams.group_by do |game|
+      game.team_id
+    end
+  end
 
-#   def highest_visitor_team
-#     best_away_team = away_games_by_team_id.max_by do |team_id, gameteam|
-#       gameteam.sum{|game1| game1.goals.to_i} / gameteam.count.to_f
-#     end.first
+  def highest_visitor_team
```

Comment:
I like how you separated this method into game_teams_manager rather than having it in the stat_tracker file